examples/tree_2d_dgsem/elixir_euler_kelvin_helmholtz_instability_fjordholm_etal.jl

```diff
@@ -33,43 +33,52 @@ function initial_condition_kelvin_helmholtz_instability_fjordholm_etal(x, t,
     # b1 = (rand(rng, m) .- 0.5) .* pi
     # b2 = (rand(rng, m) .- 0.5) .* pi
 
+    RealT = eltype(x)
     m = 10
-    a1 = [0.04457096674422902, 0.03891512410182607, 0.0030191053979293433,
-        0.0993913172320319,
-        0.1622302137588842, 0.1831383653456182, 0.11758003014101702, 0.07964318348142958,
-        0.0863245324711805, 0.18518716132585408]
-    a2 = [0.061688440856337096, 0.23000237877135882, 0.04453793881833177,
-        0.19251530387370916,
-        0.11107917357941084, 0.05898041974649702, 0.09949312336096268, 0.07022276346006465,
-        0.10670366489014596, 0.02477679264318211]
-    b1 = [0.06582340543754152, 0.9857886297001535, 0.8450452205037154, -1.279648120993805,
-        0.45454198915209526, -0.13359370986823993, 0.07062615913363897, -1.0097986278512623,
-        1.0810669017430343, -0.14207309803877177]
-    b2 = [-1.1376882185131414, -1.4798197129947765, 0.6139290513283818, -0.3319087388365522,
-        0.14633328999192285, -0.06373231463100072, -0.6270101051216724, 0.13941252226261905,
-        -1.0337526453303645, 1.0441408867083155]
-    Y1 = 0.0
-    Y2 = 0.0
+    a1 = [RealT(0.04457096674422902), RealT(0.03891512410182607),
+        RealT(0.0030191053979293433),
+        RealT(0.0993913172320319),
+        RealT(0.1622302137588842), RealT(0.1831383653456182), RealT(0.11758003014101702),
+        RealT(0.07964318348142958),
+        RealT(0.0863245324711805), RealT(0.18518716132585408)]
```

Comment:
You can also simplify this, e.g., 

```suggestion
    a1 = RealT[0.04457096674422902, 0.03891512410182607, 0.0030191053979293433,
        0.0993913172320319,
        0.1622302137588842, 0.1831383653456182, 0.11758003014101702, 0.07964318348142958,
        0.0863245324711805, 0.18518716132585408]
```